reference/Upload-API.yaml

```diff
@@ -17,11 +17,11 @@ tags:
 servers:
   - url: 'https://cloudmatrix.upload.streamamg.com'
```

Comment:
we can speak with Mark about this but I'm thinking this would be client specific similar to cloudpay, we could just route the traffic as well. so something like cloudmatix.api.streamamg.com

Reply:
Sounds good!! Fixed it.